pytorch_lightning/trainer/trainer.py

```diff
@@ -948,6 +948,12 @@ def test(
         #  Attach dataloaders (if given)
         self.data_connector.attach_dataloaders(model, test_dataloaders=test_dataloaders)
 
+        if not model_provided and ckpt_path == 'best' and self.fast_dev_run:
+            raise MisconfigurationException(
+                'You cannot execute testing when the model is not provided and `fast_dev_run=True`. '
+                'Provide model with `trainer.test(model=...)` or `trainer.test(ckpt_path=...)`'
+            )
+
```

Comment:
Please, move this into `__load_ckpt_weights`

That way, it is run also for `trainer.validate()`

Reply:
Done.